build-system/tasks/report-test-status.js

```diff
@@ -139,7 +139,7 @@ async function postReport(type, action) {
         // Do not use `json: true` because the response is a string, not JSON.
       });
 
-      log('Reported', cyan(`${type}/${action}`, 'to GitHub'));
+      log('Reported', cyan(`${type}/${action} to GitHub`));
```

Comment:
The intent was for `to GitHub` not to be colored.

```suggestion
      log('Reported', cyan(`${type}/${action}`), 'to GitHub');
```